src/requireAuth.ts

```diff
@@ -32,16 +33,18 @@ function getAuthClient(config: GoogleAuthOptions): GoogleAuth {
  * @param options CLI options.
  * @param authScopes scopes to be obtained.
  */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
 async function autoAuth(options: any, authScopes: string[]): Promise<void> {
   const client = getAuthClient({ scopes: authScopes, projectId: options.project });
   const token = await client.getAccessToken();
-  api.setAccessToken(token);
+  api.setAccessToken(token || "");
```

Comment:
What's up with this change?  Should this be done inside the setter function instead of when its called?